DataRepo/templates/DataRepo/study_detail.html

```diff
@@ -41,110 +41,16 @@ <h4>Study Record - {{ study.name }}</h4>
-                <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="Infusion-Rate">Infusion Rate (ul/min/g)</th>
-                <th data-filter-control="input" data-sortable="true" data-field="Tissue">Tissue</th>
-                <th data-filter-control="input" data-sortable="true" data-field="Sample">Sample</th>
-                <th data-filter-control="select" data-sortable="true" data-field="Sample-Owner">Sample Owner</th>
-                <th data-filter-control="input" data-sortable="true" data-field="Sample-Date">Sample Date</th>
-                <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="Collect-Time-Minutes">Sample Collect Time(m)</th>
-                <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="MSRun-Detail">MSRun Detail</th>
-            </tr>
-        </thead>
-        <tbody>
-            {% for i in df %}
-            <tr>
-                <td>{{ i.study }}</td>
-                <td>{{ i.study_description }}</td>
-                <td><a href="{% url 'animal_detail' i.animal_id %}">{{ i.animal }}</a></td>
-                <td>{{ i.genotype }}</td>
-                <td>{{ i.body_weight }}</td>
-                <td>{{ i.age |duration_iso_to_weeks }}</td>
-                <td>{{ i.sex }}</td>
-                <td>{{ i.diet }}</td>
-                <td>{{ i.feeding_status }}</td>
-                <!--handle None value for treatement-->
-                <td>
-                {% if i.treatment %}
-                    <a href="{% url 'protocol_detail' i.treatment_id %}">{{ i.treatment }}</a>
-                {% else %}
-                    None
-                {% endif %}
-                <td>
-                    {% if i.infusate_id %}
-                        <a href="{% url 'infusate_detail' i.infusate_id %}">{{ i.infusate_name }}</a>
-                    {% else %}
-                        None
-                    {% endif %}
-                </td>
-                <td>{{ i.labeled_elements |join:"," }}</td>
-                <td>{{ i.tracer_infusion_rate }}</td>
-                <td><a href="{% url 'tissue_detail' i.tissue_id %}">{{ i.tissue }}</a></td>
-                <td><a href="{% url 'sample_detail' i.sample_id %}">{{ i.sample }}</a></td>
-                <td>{{ i.sample_owner }}</td>
-                <td>{{ i.sample_date |convert_iso_date }}</td>
-                <td>{{ i.sample_time_collected |duration_iso_to_mins }}</td>
-                <td>
-                {% if i.msrun_id %}
-                    <a href="{% url 'msrun_detail' i.msrun_id %}">MSRun Detail</a>
-                {% else %}
-                    None
-                {% endif %}
-                </td>
-            </tr>
-            {% endfor %}
-        </tbody>
-    </table>
-    {% endif %}
+    <h6>Animal and Sample Data for the Study</6>
```

Comment:
Blocking: This is what is causing the bold font throughout the table.

```suggestion
    <h6>Animal and Sample Data for the Study</h6>
```

Reply:
@lparsons Nice catch! fixed now.